spynnaker/pyNN/models/utility_models/synapse_expander/synapse_expander.py

```diff
@@ -36,9 +36,11 @@
 
 
 def synapse_expander(
-        app_graph, graph_mapper, placements, transceiver,
-        provenance_file_path, executable_finder):
-    """ Run the synapse expander - needs to be done after data has been loaded
+        app_graph, placements, transceiver, provenance_file_path,
+        executable_finder):
+    """ Run the synapse expander - needs to be done after data has been loaded.
+
+    Note that this is an algorithm.
```

Comment:
Fair enough that this is an algorithm, yes, but a reason as to why this has been noted here is probably a good idea.

Reply:
I noted it because it looks like a function, as opposed to being a callable object like in most of the other algorithms in our code.

It mattered to me because it reminded me to check the XML when I changed the arguments.